README.md

```diff
@@ -197,6 +197,10 @@ On Ubuntu, need development packages and snappy library.
 
     $ sudo apt-get install build-essential autoconf automake libtool libsnappy-dev
 
+On CentOS 7 installation is also necessary.
+
+    $ sudo yum install gcc autoconf automake libtool snappy-devel
```

Comment:
I guess that we can execute `yum groupinstall "Development tools"` instead of `yum install gcc autoconf automake libtool`.
How about this?

Reply:
Thank you for reply.

I think that "Delevelopment tools" is sufficient. 
However, I think that many unnecessary packages will be installed.

In addition, "groupinstall" may not be available when using offline local repository.
Therefore, I wrote to install only the minimum necessary package.

If "Delevelopment tools" is better, we will rewrite it.

Regard.

Reply:
:+1:
I see.
It looks enough for minimum installation.

Reply:
Thank you.
Please let me know if there are anything else.